### DISCUSSION TITLE
percentage last 7th day instead of the last day

### DISCUSSION THREAD

**Discussion by 2803media** [plausible/analytics#2846] [Anything Else]
Hi one feature which can be very interesting is the possibility to change the percentage of the last day to the last 7th day. Indeed it's more logical to compare monday to monday and not monday to sunday

**Comment by metmarkosaric** [comment]
hi @2803media! we're finalizing a comparison feature which will allow you to compare to what you want. see a work in progress screenshot here: https://twitter.com/PlausibleHQ/status/1646057472396873728. hoping to release it to all later this month. stay tuned!

**Reply by 2803media** [reply]
Great indeed! I was talking about the defaut % on the chart which is quite a different approach:

![Screenshot 2023-04-13 at 11 59 19](https://user-images.githubusercontent.com/642545/231724953-450f0b95-3c12-4861-a922-f84baa1d1a1a.jpg)

For me the % of the last day is quite irrelevant as you compare different kind of days ;)

A possibility to change this in the site setting should be very cool

**Reply by metmarkosaric** [reply]
these percentages will be going away after we introduce a fully featured comparison. they're there mostly because we don't have a comparison feature but when we introduce it, it will be way better and more flexible to use the new feature for any comparisons

**Comment by metmarkosaric** [comment]
**Update:** You can now compare your stats over time! Select "**Compare**" in the date picker or press the "**X**" key on your keyboard to start comparing. Learn more about the comparisons here: https://plausible.io/docs/compare-stats.
